### ISSUE TITLE
Leukocyte migration defect

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11108]
I do not understand the suggestion:
replaced by Decreased neutrophil adhesion
these are not fully synonymous?

**Comment by adamklocperk** [comment]
They are not fully synonymous, as there are other leukocyte than neutrophils, and migration can be different than just adhesion. Nevertheless, it is our opinion that the commonly described and "routinely" evaluated feature is neutrophil adhesion. For this reason we propose to change to wording to Decreased neutrophil adhesion. This should not lead to any confusion, as there are not annotations using HP:0033797 or HP:0033798 at present, so we will not mis-label any diseases by changing the term from "leukocyte migration" to "neutrophil adhesion".

**Comment by pnrobinson** [comment]
@adamklocperk 
There is currently a subterm, [Impaired leukocyte adhesion HP:0033798](https://hpo.jax.org/browse/term/HP:0033798).
So just to confirm- we shoud merge this and rename it Decreased neutrophil adhesion
Do you happen to knof of a good reference? I could onlöy find this: https://pubmed.ncbi.nlm.nih.gov/2615675/

**Comment by adamklocperk** [comment]
@pnrobinson strangely, I found that terms related to granulocyte function (= physiology), such as adhesion capabilities, oxidative burst capacity, etc, exist, such as [Abnormality of neutrophil physiology HP:0011990](https://hpo.jax.org/browse/term/HP:0011990), but are located under Abnormal granulocyte morphology. This is weird to me, I thought of these two branches (morphology, physiology) as separate. This also throws a wrench into the plan we made in Berlin in November with our group. Here is my improvisation to make this work.

**I think [Abnormality of neutrophil physiology HP:0011990](https://hpo.jax.org/browse/term/HP:0011990) should be made a child of [Abnormal leukocyte physiology](https://hpo.jax.org/browse/term/HP:0033796).**

**Decreased neutrophil adhesion should be a child of [Abnormality of neutrophil physiology HP:0011990](https://hpo.jax.org/browse/term/HP:0011990)** (which already exists), or the Abnormal granulocyte physiology (which we proposed) - these are just different spellings, we used Abnormal XYZ to keep things consistent with other similar terms - use one of them.

[Leukocyte migration defect HP:0033797](https://hpo.jax.org/browse/term/HP:0033797) and [Impaired neutrophil chemotaxis HP:0040238](https://hpo.jax.org/browse/term/HP:0040238) should then be merged into/renamed Decreased neutrophil adhesion - all describe the same thing, in my opinion. A good reference would be https://www.omim.org/entry/116920, of course, but for individual papers, https://rupress.org/jem/article-abstract/160/6/1901/49410/Inherited-deficiency-of-the-Mac-1-LFA-1-p150-95?redirectedFrom=fulltext.

[Abnormal L-selectin shedding HP:0410251](https://hpo.jax.org/browse/term/HP:0410251) is the same thing as our "Decreased TLR signaling" - we prefer our name, but they should be largely the same thing and should be siblings of Decreased neutrophil adhesion, children of [Abnormality of neutrophil physiology HP:0011990](https://hpo.jax.org/browse/term/HP:0011990).

[Impaired oxidative burst HP:0003203](https://hpo.jax.org/browse/term/HP:0003203) is the same as our "Decreased neutrophil oxidative burst". They should be merged and I have no clear preference for either name.

[Impaired neutrophil killing of staphylococci HP:0005512](https://hpo.jax.org/browse/term/HP:0005512) I believe to be superfluous and the disease associations should be changed to [Impaired oxidative burst HP:0003203](https://hpo.jax.org/browse/term/HP:0003203).

Sorry this is a bit long, but it took me some work, because there appeared this whole unexpected branch with overlapping terms, in a location we were not aware of...

**Comment by adamklocperk** [comment]
Revising the neutrophil migration/adhesion/chemotaxis topic will require input from other experts. For now, I would perhaps preserve the old terms and associations and we can decide on the structure with our group.
